### PR TITLE
card

### PR DESCRIPTION
**Pull request by Mandhularajitha**

### CODE REVIEW

File: cards.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    margin-top: 2rem;
+}
+
+.product {
+    margin: 20px;
+    width: 400px;
+    height: 550px;
+    box-shadow: 10px 10px 40px rgb(98, 122, 122);
+    padding: 70px;
+    background-color: rgb(117, 249, 253);
```
Comment (samflab) [inline cards.css:25]: use variables for colours

File: cards.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+.card-container {
+    display: flex;
+    margin-top: 2rem;
+}
+
+.product {
+    margin: 20px;
+    width: 400px;
+    height: 550px;
```
Comment (samflab) [inline cards.css:22]: use rem instead of px

File: cards.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    flex-wrap: wrap;
+}
+
+.product_image {
+    width: 100%;
+
+}
+
+.product_title {
+    font-size: 20px;
```
Comment (samflab) [inline cards.css:36]: font size will also be in rem

File: cards.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+.product_basket:hover {
+    background-color: green;
+    color: white;
+}
+
+.product_buy {
+    background-color: dodgerblue;
+    color: white;
```
Comment (samflab) [inline cards.css:86]: use css variables and not colour names

File: cards.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Arial;
+    box-sizing: border-box;
+}
+
```
Comment (samflab) [inline cards.css:7]: put reset.css in another file

File: cards.html [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+            </div>
+            <img src="card2.jpeg" alt="shoes" class="product_image">
+
+            <button class="product_button product_basket">Add to Basket</button>
+            <button class="product_button product_buy">Buy Now</button>
+
+        </div>
+
+        <div class="product">
+            <p class="product_tittle">Diamond Ring</p>
```
Comment (samflab) [inline cards.html:76]: here title spelling is wrong

**Review by samflab — CHANGES_REQUESTED** [review]

**Comment by samflab** [comment]
Deploy your project on Netlify. It's easier to see the output. 
Please add a description and screenshots to your PR

File: cards.html [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="cards.css" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <script src="https://kit.fontawesome.com/8ddd15368f.js"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="cards.css">
+    <title>Document</title>
```
Comment (rutvikpumak) [inline cards.html:12]: change title

File: cards.css [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+.product_button {
+    width: 100%;
+    height: 40px;
+    font-size: 18px;
+    cursor: pointer;
+    margin-top: 10px;
+    border: none;
```
Comment (rutvikpumak) [inline cards.css:69]: use common reset css for button

File: cards.css [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+    outline: none;
+    transition: 0.3s;
+}
+
+.product_basket {
+    background-color: greenyellow;
```
Comment (rutvikpumak) [inline cards.css:75]: use css variable for color

**Comment by rutvikpumak** [comment]
Add all images in a common folder

File: cards.html [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+            <p class="product_tittle">Diamond Ring</p>
+            <p class="product_price">
+                <small><i class="fa-solid fa-indian-rupee-sign"></i></small>
+                <span>1699</span>
+            </p>
+            <div class="product_rating">
+                <p><i class="fa fa-star" aria-hidden="true"></i></p>
+                <p><i class="fa fa-star" aria-hidden="true"></i></p>
+                <p><i class="fa fa-star" aria-hidden="true"></i></p>
+                <p><i class="fa fa-star" aria-hidden="true"></i></p>
```
Comment (mohammadkhizerkhan) [inline cards.html:85]: add 5 stars

**Review by Sindhu983 — APPROVED** [review]

**Review by NamrataRaikwar2002 — APPROVED** [review]

**Pull request merged**